Documentation/gettingstarted/aws.rst

```diff
@@ -0,0 +1,301 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+****************************************
+Getting Started Securing AWS Instances
+****************************************
+
+This document serves as an introduction to using Cilium to enforce policies
+based on AWS instances metadata. It is a detailed walk-through of getting a
+single-node Cilium environment running on your machine. It is designed to take
+15-30 minutes with some experience running Kubernetes.
+
+.. include:: minikube_intro.rst
+
+Step 1: Create AWS secrets
+===========================
+
+Before install Cilium, a new Kubernetes Secret with the AWS Tokens needs to be
+in place, that secret will allow Cilium to gather the info from the Amazon AWS
```

Comment:
Run-on sentence.

"... needs to exist in your Kubernetes cluster. This secret will allow..."